### ISSUE TITLE
Remove document panel from VoiceOver hierarchy

### ISSUE THREAD

**Issue by edgm74** [edgm74/chorustr-community#256]
Currently voiceover includes PDF documents and notes within the navigation hierarchy for the app.  The document panel should be excluded because visually impaired users don't tend to consume documents via the app.
